OsmAnd/src/net/osmand/plus/configmap/tracks/TracksFragment.java

```diff
@@ -267,11 +287,11 @@ private void setupButtons(@NonNull View view) {
 	}
 
 	private void updateButtonsState() {
-		boolean anySelected = !Algorithms.isEmpty(selectedTracksHelper.getSelectedTracks());
+		boolean anySelected = !Algorithms.isEmpty(itemsSelectionHelper.getSelectedItems());
```

Comment:
Do not create new array to check if empty! 
Review all pull-request and drop fix similar code.